params/precompile_config_test.go

```diff
@@ -7,19 +7,21 @@ import (
 	"math/big"
 	"testing"
 
+	"github.com/ava-labs/subnet-evm/commontype"
 	"github.com/ava-labs/subnet-evm/precompile"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
 func TestValidateWithChainConfig(t *testing.T) {
```

Comment:
should we rename to `TestVerify` or `TestVerifyChainConfig`?

Reply:
done